whois-endtoend/src/test/groovy/net/ripe/db/whois/spec/integration/OverrideIntegrationSpec.groovy

```diff
@@ -263,43 +263,70 @@ class OverrideIntegrationSpec extends BaseWhoisSourceSpec {
                 "***Info:    Authorisation override used")
     }
 
-    def "override is noop on case-sensitive change with update-on-noop set to false"() {
+    def "add comment is not a noop for whois update"() {
         given:
-        def data = fixtures["ORG1"].stripIndent() + "override:denis,override1, {update-on-noop=false}"
-        data = (data =~ /org-name:     Test Organisation Ltd/).replaceFirst("org-name: test organisation ltd")
+        def data = fixtures["ORG1"].stripIndent() + "override:denis,override1"
+        data = (data =~ /address:      street 5/).replaceFirst("address:      street 5 #test comment noop")
 
         def update = new SyncUpdate(data: data)
 
         when:
         def result = syncUpdate update
 
         then:
-        result.contains("Warning: Submitted object identical to database object\n" +
+        !result.contains("Warning: Submitted object identical to database object\n" +
+                "\n" +
+                "***Info:    Authorisation override used")
+        result.contains("Modify SUCCEEDED: [organisation] ORG-TOL1-TEST\n" +
+                "\n" +
                 "\n" +
                 "***Info:    Authorisation override used")
     }
 
-    def "override on case-sensitive change with update-on-noop set to true"() {
```

Comment:
Was this test removed?

Reply:
No, test case still exists, I think I changed few lines above so somehow it shows deleted

Reply:
Ahh, I added new tests cases but since lot of lines are similar it treated it as changed , look a. little bit down you will find this test as an addition